dev/doc/econstr.md

```diff
@@ -0,0 +1,128 @@
+# Evar-insensitive terms (EConstr)
+
+Evar-insensitive terms were introduced in 8.7, following
+[CEP #10](https://github.com/coq/ceps/blob/master/text/010-econstr.md). We will
+not recap the motivations in this document and rather summarize the code changes
+to perform.
+
+## Overview
+
+The essential datastructures are defined in the `EConstr` module. It defines
```

Comment:
It would be nice to link to the file here as such:

     in [the `EConstr` module](/engine/eConstr.mli)

Obviously most plugin developers already know where to look for this kind of stuff but new ones do not necessarily and it is nice to guide them as much as possible.
Plus, the link will show on GitHub (which is the most suitable interface for reading Markdown files), so one can quickly go from the doc to the file (as it is the case in [the XML protocol doc](https://github.com/coq/coq/blob/trunk/dev/doc/xml-protocol.md)).